packages/indexer/README.md

```diff
@@ -58,6 +58,27 @@ for await (const tx of txCollector.collect()) {
 }
 ```
 
+Query transactions between given `block_number` range is supported:
+
+```javascript
+txCollector = new TransactionCollector(indexer, {
+  lock: {
+    code_hash: 
+      "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8",
+    hash_type: "type",
+    args: "0xa528f2b9a51118b193178db4cf2f3db92e7df323",
+  },
+  fromBlock: 1000,
+  toBlock: 10000,
+});
```

Comment:
As the lock script showed in original doc is only appeared in genesis block so far, which is too trivial for a range query demo, so here replace it with cell base output lock script which  appeared in over 290k transactions.